src/common/Checkbox.svelte

```diff
@@ -25,14 +25,14 @@
 </script>
 
 <style lang="scss">
-  $checksize: 15px;
-  $checklinesize: 11px;
-  $xOff: 0;
-
   label {
+    --checksize: 15px;
+    --checklinesize: 11px;
+    --xOff: 0;
+
     display: inline-block;
-    height: $checkboxSize;
-    width: $checkboxSize;
+    height: var(--checkboxSize, 22px);
```

Comment:
I have not used CSS variables extensively, but putting default values in for all of the uses seems like it defeats the purpose of using variables.

Reply:
It's a little insurance in case I forget to set one. It's not required, but I'm being careful while we're migrating.